projects/observability/src/shared/dashboard/data/graphql/table/entity/entity-table-data-source.model.ts

```diff
@@ -59,17 +68,38 @@ export class EntityTableDataSourceModel extends TableDataSourceModel {
     filters: GraphQlFilter[],
     request: TableDataRequest<SpecificationBackedTableColumnDef>
   ): GraphQlEntitiesQueryRequest {
+    // tslint:disable-next-line: strict-boolean-expressions
+    if (this.isClientSideRendered) {
```

Comment:
Isn't the whole app client side rendered? Did you mean `isClientSideFiltered`?

Reply:
Yeah, makes sense, I'll rename the var.